InscryptionCommunityPatch/Card/PackMuleBugFix.cs

```diff
@@ -0,0 +1,19 @@
+using DiskCardGame;
+using HarmonyLib;
+
+namespace InscryptionCommunityPatch.Card
+{
+    //This is a bugfix to the packmule special ability, to prevent it from softlocking if the player some how obtains it and tries to play it.
+    public class PackMuleBugFix
```

Comment:
Make the class internal static and all patches private so it won't show up in the documentation

Reply:
done